subsys/bluetooth/shell/bt.c

```diff
@@ -1833,6 +1871,15 @@ static int cmd_l2cap_send(int argc, char *argv[])
 	return 0;
 }
 
+static int cmd_l2cap_recv(int argc, char *argv[])
+{
+	if (argc > 1) {
+		l2cap_recv_delay = strtoul(argv[1], NULL, 10);
+	}
```

Comment:
else recv_delay = 0

Reply:
I prefer to have it print the current delay if no parameter is given.